exec/java-exec/src/main/java/org/apache/drill/exec/store/pojo/PojoDataType.java

```diff
@@ -48,6 +49,11 @@ public PojoDataType(Class<?> pojoClass) {
       Class<?> type = f.getType();
       names.add(f.getName());
 
+      //Look up @Nullability for nullable property
+      Nullability nullability = f.getDeclaredAnnotation(Nullability.class);
+      nullables.add(nullability == null ? //Absence of annotation => (isNullable=true)
```

Comment:
`f.isAnnotationPresent(Nullability.class)` returns a `boolean` result; may simplify the code here

Reply:
It seemed very verbose, but that makes sense. I'll make this change.